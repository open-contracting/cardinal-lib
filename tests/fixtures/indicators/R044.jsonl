{"ocid":"F1","parties":[{"address":{"streetAddress":"Street ABC","locality":"Atlantis City","region":"Moorland","postalCode":"1234","countryName":"Atlantis"},"roles":["tenderer"],"id":"GB-COH-09506232"},{"address":{"streetAddress":"Street ABC","locality":"Atlantis City","region":"Moorland","postalCode":"1234","countryName":"Atlantis"},"roles":["tenderer"],"id":"GB-COH-09506234"}]}
{"ocid":"F2","parties":[{"address":{"streetAddress":"Street ABC"},"roles":["tenderer"],"id":"GB-COH-09506232"},{"address":{"streetAddress":"Street ABC"},"roles":["tenderer"],"id":"GB-COH-09506234"}]}
{"ocid":"Same country only","parties":[{"address":{"countryName":"Atlantis"},"roles":["tenderer"],"id":"GB-COH-09506232"},{"address":{"countryName":"Atlantis"},"roles":["tenderer"],"id":"GB-COH-09506234"}]}
{"ocid":"No address","parties":[{"roles":["tenderer"],"id":"GB-COH-09506232"},{"roles":["tenderer"],"id":"GB-COH-09506234"}]}
