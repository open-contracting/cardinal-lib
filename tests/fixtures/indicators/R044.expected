{"OCID":{"F1":{"R044":1.0},"F2":{"R044":1.0}},"Tenderer":{"GB-COH-09506234":{"R044":1.0},"GB-COH-09506232":{"R044":1.0}}}